CONTRIBUTING.md

```diff
@@ -177,6 +177,23 @@ To maintain understandable code and avoid circular dependencies it is important
 - **templates:** Golang templates for generating the html output.
 - **vendor:** External code that Gitea depends on.
 
+## API v1
+
+The API is documented by [swagger](http://gitea.com/api/swagger) and is based on [GitHub API v3](https://developer.github.com/v3/).
+So if there exist a endpoint on GitHub API it should be the same for gitea. If GitHub dont provide a functionality Gitea does, a own endpoint can cover this. If a information is on Gitea but not on GitHub API a new field is used wich doesnt coldie with GitHub ones.
```

Comment:
Grammar typo.

Fix:
```suggestion
So if there exists a endpoint on GitHub API it should be the same for gitea. If GitHub don't provide a functionality Gitea does, an own endpoint can cover this. If an information is on Gitea but not on GitHub API a new field is used which doesn't collide with GitHub one.
```

Reply:
in suggestion 
1. `GitHub don't provide`-> `GitHub doesn't provide`
2. `an own endpoint` -> `an new endpoint`
3. `If an information is` -> `If information is`